requirements.txt

```diff
@@ -6,6 +6,7 @@ cachetools==2.0.1
 configargparse==0.13.0
 humanfriendly==4.16.1
 psycopg2-binary==2.7.5
+requests==2.21.0
```

Comment:
This should be removed

Reply:
Not sure why you get this, but to me it appears it is already removed
https://github.com/src-d/lookout-sdk-ml/pull/89/commits/4e0a549fba73e74787a91c7ef9188bfffddbc2fd